cairo/ethereum_rlp/rlp.cairo

```diff
@@ -242,9 +240,56 @@ func Extended__eq__(left: Extended, right: Extended) -> bool {
         return res;
     }
 
-    with_attr error_message("Unimplemented for types other than bytes and sequence") {
-        jmp raise.raise_label;
+    // Bytearray case
+    if (left.value.bytearray.value != 0) {
+        if (right.value.bytearray.value != 0) {
+            let res = Bytes__eq__(left.value.bytearray, right.value.bytearray);
+            return res;
+        }
+        let res = bool(0);
+        return res;
     }
+
+    // Bytes case
+    if (left.value.bytes.value != 0) {
```

Comment:
same as above if you agree

Reply:
if we want to early return 'smartly', we could do similarly as with `TransactionType`:
- `get_transaction_type`

e.g. - get_extended_type(value: Extended) -> felt 


but it doesn't seem important enough to change